.github/workflows/arm-incremental-typespec-preview.yaml

```diff
@@ -29,6 +28,16 @@ jobs:
           sparse-checkout: |
             .github
 
+      - name: Setup Node 20 and install deps
+        uses: ./.github/actions/setup-node-install-deps
+        with:
+          # actions/github-script@v7 uses Node 20
+          node-version: 20.x
+          # "--no-audit": improves performance
+          # "--omit dev": not needed at dev time, improves performance
+          install-command: "npm ci --no-audit --omit dev"
```

Comment:
Should the action use these params by default?

Reply:
It defaults to {"22.x", "npm ci", and "repo root"}, since this is better for things like eng/tools.  If we need to call this from more workflows, we could create **another** reusable action with these defaults, but it seemed like overkill to me right now.